great_expectations/datasource/data_connector/configured_asset_sql_data_connector.py

```diff
@@ -60,13 +62,37 @@ def assets(self) -> Dict[str, dict]:
 
     def add_data_asset(
         self,
-        name: str,
-        config: dict,
+        data_asset_name: str,
+        data_asset_config: dict,
     ):
         """
         Add data_asset to DataConnector using data_asset name as key, and data_asset configuration as value.
         """
-        self._assets[name] = config
+        data_asset_name = self._update_data_asset_name_from_config(
+            data_asset_name, data_asset_config
+        )
+        self._assets[data_asset_name] = data_asset_config
+
+    def _update_data_asset_name_from_config(
+        self, data_asset_name: str, data_asset_config: dict
+    ) -> str:
+
+        data_asset_name_prefix = data_asset_config.get("data_asset_name_prefix", "")
+        data_asset_name_suffix = data_asset_config.get("data_asset_name_suffix", "")
+        schema_name = data_asset_config.get("schema_name", "")
+        include_schema_name = data_asset_config.get("include_schema_name", "")
+        if schema_name and include_schema_name is False:
```

Comment:
If `include_schema_name` is a bool, can we set the default return from `get` to a bool as well?

Reply:
Yes. This should be `True`. Thank you for catching this @cdkini